src/core/meson.build

```diff
@@ -197,7 +197,11 @@ libcore = shared_library(
 
 core_includes = [includes, include_directories('.')]
 
-systemd_sources = files('main.c')
+systemd_sources = files(
+        'main.c',
+        'main.h',
+        'crash-handler.c',
+        'crash-handler.h')
```

Comment:
In a recent meson PR by @keszybz we bikeshedded to put the paren on a newline for calls to files() so we can use a  trailing comma which makes future diffs smaller so let's do that here as well.